pyproject.toml

```diff
@@ -30,10 +28,10 @@ api_version = "v1"
 # or other special characters because it will be used in an import statement.
 package_name = "invest_demo_plugin"
 
-# List of conda dependencies to install into the plugin's conda env 
-# Only include dependencies that cannot be specified in project.dependencies because 
-# they are not available thru pip. Usually this is only python itself, and any 
-# non-python dependencies like the GDAL binaries.
+# List of conda dependencies to install into the plugin's conda env. 
+# Include dependencies that can be installed from conda-forge.
```

Comment:
I don't know that I agree we should be giving this guidance. I would maybe change it to a neutral position - something like "Including dependencies here may help resolve build issues".